internal/cmd/keys_test.go

```diff
@@ -92,19 +92,13 @@ func TestKeysAddCmd(t *testing.T) {
 		ch := setup(t)
 
 		ctx, bufs := PatchCLI(context.Background())
-		err := Run(ctx, "keys", "add", "--ttl=400h", "--extension-deadline=5h", "my-user")
+		err := Run(ctx, "keys", "add", "--ttl=400h", "--extension-deadline=5h", "--user=my-user")
 		assert.NilError(t, err)
 
 		req := <-ch
 		assert.Equal(t, req.Name, "") // filled by server
 		assert.Equal(t, withNewline(bufs.Stdout.String()), expectedKeysAddOutput)
 	})
-
-	t.Run("without required arguments", func(t *testing.T) {
```

Comment:
This is still a good test case, we can change it to "with unexpected arguments" to show that it should fail if you give it any positional arguments.  I believe that will fail right now since `Args: NoArgs` is missing.

I think it would be great to add tests cases for the other parameter combinations (ex: no `--user` flag), but maybe we should convert this test to use a real API sever first. Would be fine in a follow up.